README.fr.md

```diff
@@ -39,21 +33,21 @@ IRC ou envoyez un e-mail à sir@cmpwn.com (en anglais seulement) pour des consei
 
 Installez les dépendances :
 
-* meson
-* [wlc](https://github.com/Cloudef/wlc)
+* meson \*
+* [wlroots](https://github.com/swaywm/wlroots)
 * wayland
-* xwayland
-* libinput >= 1.6.0
-* libcap
+* wayland-protocols \*
 * pcre
-* json-c >= 0.13
+* json-c
 * pango
 * cairo
-* gdk-pixbuf2 *
-* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (requis pour les pages man)
-* git
+* gdk-pixbuf2 \*\*
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) (optionnel: requis pour les pages man) \*
+* git \*
+
+_\*Requis uniquement pour la compilation_
 
-_\*Uniquement requis pour swaybar, swaybg_
+_\*\*optionnel: requis uniquement pour swaybg_
```

Comment:
Missing a capital letter here

Reply:
This error is also present in the english readme. We should probably fix it!

Reply:
Please do!